<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>278</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>321</width>
    <height>527</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dry Eye Syndrome Self Diagnosis</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Image/Dry Eye Syndrome Self Diagnosis icon.jpg</normaloff>../Image/Dry Eye Syndrome Self Diagnosis icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Image/Back.png</normaloff>../Image/Back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>53</width>
     <height>39</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>321</width>
     <height>51</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>321</width>
     <height>51</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>321</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color:rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string>I-safety</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(211, 211, 211);
background-color:rgb(255, 255, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>version:1.2.12 </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(211, 211, 211);
background-color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>email: fix91190@gmail.com</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(211, 211, 211);
background-color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>developer: Bang jeho, Jeon sejong</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>311</width>
     <height>20</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>311</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>311</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬돋움</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:0px;</string>
   </property>
   <property name="text">
    <string>Catch eye Blink                                        &gt;</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Image/cute_cow-07-512.png</normaloff>
     <normalon>../Image/cute_cow-04-512.png</normalon>
     <disabledoff>../Image/cute_cow-07-512.png</disabledoff>
     <disabledon>../Image/cute_cow-04-512.png</disabledon>
     <activeoff>../Image/cute_cow-07-512.png</activeoff>
     <activeon>../Image/cute_cow-04-512.png</activeon>
     <selectedoff>../Image/cute_cow-07-512.png</selectedoff>
     <selectedon>../Image/cute_cow-04-512.png</selectedon>../Image/cute_cow-07-512.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>41</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬돋움</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
   border:0px;
   background-color:white;
}
QPushButton:hover
{
   background-color:rgb(241, 241, 241);
}
</string>
   </property>
   <property name="text">
    <string>Self Survey                                                &gt; </string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Image/1.jpg</normaloff>../Image/1.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>41</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>311</width>
     <height>20</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>311</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>311</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_8</zorder>
  <zorder>label_11</zorder>
  <zorder>label_10</zorder>
  <zorder>line_8</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>line_7</zorder>
  <zorder>pushButton_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
